SVGKitLibrary/Demo-OSX/en.lproj/Credits.rtf

```diff
@@ -0,0 +1,29 @@
+{\rtf0\ansi{\fonttbl\f0\fswiss Helvetica;}
```

Comment:
Either delete this file or have it direct people straight to the github repository URL?